datafusion/core/src/physical_planner.rs

```diff
@@ -208,10 +208,15 @@ fn create_physical_name(e: &Expr, is_first_expr: bool) -> Result<String> {
                     let key = create_physical_name(key, false)?;
                     format!("{expr}[{key}]")
                 }
-                GetFieldAccess::ListRange { start, stop } => {
+                GetFieldAccess::ListStride {
```

Comment:
One small thing, I think stride is equivalent to step, IMO ListRange is more appropriate than ListStride